Articles/Blog/what-is-an-inclusive-minute.md

```diff
@@ -0,0 +1,143 @@
+We have shared inclusive minutes about people ([Lynne Ann Conway](https://en.wikipedia.org/wiki/Lynn_Conway), [Gladys West](https://en.wikipedia.org/wiki/Gladys_West)), about films ([Picture a Scientist](https://en.wikipedia.org/wiki/Picture_a_Scientist), [Coded Bias](https://en.wikipedia.org/wiki/Coded_Bias)), about resources ([inclusive clip art](https://diversityandinclusion.uchicago.edu/resources/inclusive-illustrations/), [automated closed captioning](https://www.ai-media.tv/ai-media-blog/the-best-free-captioning-and-subtitling-tools-2/), [inclusive design](https://inclusive.microsoft.design)), about team meetings ([interruptions](https://www.wnycstudios.org/podcasts/radiolabmoreperfect/episodes/justice-interrupted), [quite-tude for introverts](https://susancain.net/book/quiet/)) about recruiting ([ECP BE program](https://science.osti.gov/-/media/ascr/ascac/pdf/meetings/202203/ASCAC_202203_ECP_BroaderEngagement_Overview.pdf), expanding networks to include MSI/HBCUs) and more.
+
+#### Example inclusive minutes
+
+Lets suppose your team is an HPC/CSE software development team.
+In your case **X** is HPC/CSE software development.
+Examples of inclusive minute topics that are, by the terms offered here, within scope include...
+
+* A short description of the book [Black Software](https://en.wikipedia.org/wiki/Black_Software) and key take-aways you learned from reading it.
+* A reminder that color choices in UI design impact color-blind users.
+* Relaying a recent conversation with a colleague that feels Fortran programmers aren’t sufficiently supported in HPC/CSE.
+
+Examples of inclusive minute topics that are, by the terms offered here, out of scope include...
+
+* Sharing a recent personal negative interaction with a security official at the airport.
+  * Is it appropriate length? Probably. It certainly would not take more than a minute or two to share this experience.
+  * Is it closely related to **X**? Unless **X** indeed involves airport security, it probably is not.
+* Playing the YouTube video [Equality, Diversity and Inclusion](https://www.youtube.com/watch?v=Q6eqOBOqESM)
+  * Is it appropriate length? No. Its 6 minutes. 2-3 minutes might work but 6 is a bit too long.
+  * Is it closely related to **X**? Probably not. It is more of a general resource regarding inclusion and does not seem to tie inclusion into any specific **X**.
+* Going page-by-page through a resource such as [Microsoft's Inclusive Design](https://inclusive.microsoft.design/tools-and-activities/Inclusive101Guidebook.pdf).
+  * Is it appropriate length? The resource, by itself, is 50+ pages and would take too long to go through. However, sharing a link to the article and two or three take-aways in the form of a single slide would be appropriate length.
+  * Is it closely related to **X**? Even if **X** is software development, maybe not. The resource is extremely high level and general and really doesn't tie inclusion topics with an **X**.
+
+#### What is inclusion?
```

Comment:
This probably needs to go up higher. I suspect you put it lower so the true "meat" of the article can be read first, but you might lose people right away if you don't at least give them an idea of what you mean by "inclusion." Maybe put a small definition at the top and then say, "More details given later" or something to that effect.

Reply:
I'll come back to this.

Reply:
Addressed (hopefully satisfactorily) in eada8259e19a5c45b08f4a591f4398a9d5a472fb